validator/src/main/resources/expected-data-template/node/eks/outgoing-http-call-metric.mustache

```diff
@@ -0,0 +1,227 @@
+-
+  metricName: Latency
+  namespace: {{metricNamespace}}
+  dimensions:
+    -
+      name: Operation
+      value: GET /outgoing-http-call
+    -
+      name: Service
+      value: {{serviceName}}
+    -
+      name: Environment
+      value: eks:{{platformInfo}}/{{appNamespace}}
+
+-
+  metricName: Latency
+  namespace: {{metricNamespace}}
+  dimensions:
+    -
+      name: Operation
+      value: GET /outgoing-http-call
+    -
+      name: Service
+      value: {{serviceName}}
+    -
+      name: Environment
+      value: eks:{{platformInfo}}/{{appNamespace}}
+    -
+      name: RemoteOperation
+      value: GET /
+    -
+      name: RemoteService
+      value: www.amazon.com:80
```

Comment:
Why is this :80 here?

Reply:
We made HTTP calls which default to port :80, an HTTPS call defaults to :443. These are additions to the telemetry that simply help to know which type of request was made and are not specifically against the design of how we vend telemetry in ADOT in any language. For example, Java may not add the port for HTTP requests that do not have a specified port (default to 80), while JavaScript requests append this no matter what making it appear in the telemetry.